cpt/runner.py

```diff
@@ -113,7 +113,7 @@ def run(self):
                             return
                         for installed in r['installed']:
                             reference = installed["recipe"]["id"]
-                            if ((reference == str(self._reference)) or \
+                            if ((str(self._reference) in reference) or \
```

Comment:
Weak. Use ConanfileReference.load(ref).copy_clear_rev() to compare. Search examples in Conan code if needed.

Reply:
Done!